vendor/plist_to_html/plist_to_html/dist/js/bugviewer.js

```diff
@@ -10,6 +10,35 @@ var BugViewer = {
   init : function (files, reports) {
     this._files = files;
     this._reports = reports;
+
+    this.initEscapeChars();
+  },
+
+  initEscapeChars : function () {
+    this.escapeChars = {
+      ' ' : 'nbsp',
+      '<' : 'lt',
+      '>' : 'gt',
+      '"' : 'quot',
+      '&' : 'amp',
+      '\'' : '#39'
+    };
```

Comment:
Do we want to escape some other characters too?
```
    "'": '&#39;',
    '/': '&#x2F;',
    '`': '&#x60;',
    '=': '&#x3D;'
```

Reply:
@gyorb We don't have to escape these characters. I tested it.